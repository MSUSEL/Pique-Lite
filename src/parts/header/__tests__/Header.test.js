import Header from "../Header.component";
import {shallow} from 'enzyme';
import React from 'react';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

const initStore = {
    toggle: true
}
// a mock redux store is created 
const mockStore = configureStore();
const store = mockStore(initStore);

describe('Header Component', () => {
    const wrapper = shallow(
        <Provider store={store}>
            <Header/>
        </Provider>
    );
    test('should render correctly with default props and redux store', () => {
        expect(wrapper).toMatchSnapshot();
    });
});