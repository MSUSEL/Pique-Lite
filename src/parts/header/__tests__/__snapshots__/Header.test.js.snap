// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly with default props and redux store 1`] = `ShallowWrapper {}`;
